opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -43,6 +43,29 @@
 
 namespace Opm {
 
+	const std::vector <std::string> SummaryConfig::__ALL_expands_keywords ({
+                "FOPR", "GOPR", "WOPR", "FOPT", "GOPT", "WOPT",
+                "FOIR", "GOIR", "WOIR", "FOIT", "GOIT", "WOIT",
+                "FWPR", "GWPR", "WWPR", "FWPT", "GWPT", "WWPT",
+                "FWIR", "GWIR", "WWIR", "FWIT", "GWIT", "WWIT",
+                "FGPR", "GGPR", "WGPR", "FGPT", "GGPT", "WGPT",
+                "FGIR", "GGIR", "WGIR", "FGIT", "GGIT", "WGIT",
+                "FVPR", "GVPR", "WVPR", "FVPT", "GVPT", "WVPT",
+                "FVIR", "GVIR", "WVIR", "FVIT", "GVIT", "WVIT",
+                "FWCT", "GWCT", "WWCT", "FGOR", "GGOR", "WGOR",
+                "FWGR", "GWGR", "WWGR",
+                "WBHP", "WTHP", "WPI",
+                "FOIP", "FOIPL", "FOIPG",
+                "FWIP",
+                "FGIP", "FGIPL", "FGIPG", "FPR",
+                "FAQR", "FAQRG", "AAQR", "AAQRG",
+                "FAQT", "FAQTG", "AAQT", "AAQTG"
```

Comment:
Would be good if this was reformatted so that the `Fxxxx`, `Gxxx` and `Wxx variables were clearly separated.